Tools/glib/api_test_runner.py

```diff
@@ -317,7 +317,7 @@ def run_tests(self):
                 skipped_subtests = self._test_cases_to_skip(test)
                 number_of_total_tests += len(skipped_subtests if not subtests else set(skipped_subtests).intersection(subtests))
                 results = self._run_test(test, subtests, skipped_subtests)
-                if len(results) == 0:
+                if len(subtests) > 0 and len(results) == 0:
```

Comment:
When is `subtests <= 0`? I'm not sure I understand this change.

Reply:
The issue that is happening in TestContextMenu when GTK4 is used is that there are not subtests to run.

https://github.com/webkit/webkit/blob/main/Tools/TestWebKitAPI/Tests/WebKitGtk/TestContextMenu.cpp#L1166

Reply:
But that's because you are running only one test, right? So subtests is the subtests of that one test. Does it work if you run all the tests?

Reply:
I doubled check and `subtests` is a list of subtests passed as an argument via '-p'. `subtests` is not the group of `subtests` within a `tests`, which is what I thought.

An alternative solution is to run TestContextMenu only for GTK3. Right now none subtests are run for GTK4.